sharedUiComponents/lines/fileMultipleButtonLineComponent.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";

```

Comment:
Should this be a shared component?

Reply:
about .tsx, yup they have been excluded since day one :-( I really think they should be in as well. @RaananW you could definitely make a PR for it, but it would need to fix all the existing issues first on the already existing tools. Should not be hard but tedious as there are quite a few of them.